_pytest/pytester.py

```diff
@@ -481,6 +481,7 @@ def _makefile(self, ext, args, kwargs):
         ret = None
         for name, value in items:
             p = self.tmpdir.join(name).new(ext=ext)
+            p.dirpath().ensure_dir()
```

Comment:
This doesn't seem wrong or anything, but is it actually relevant to this code change?